packages/client/src/links/internals/contentTypes.ts

```diff
@@ -0,0 +1,15 @@
+export function isOctetType(input: unknown) {
+  return (
+    input instanceof Uint8Array ||
```

Comment:
Same story actually, all browsers have had it since prior to 2013

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array

Reply:
And Blob https://developer.mozilla.org/en-US/docs/Web/API/Blob

And File https://developer.mozilla.org/en-US/docs/Web/API/File

Same story with these, all browsers have had them more than a decade :)

Reply:
Just put in the v11 migration guide that these apis are now required so its documented somewhere

Reply:
Fair enough, it does lock us fully onto Node 18+ which wasn't the case in v10 so well worth having that in the doc

Reply:
Oh damn

I'm so fucking old